sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1050,6 +1049,7 @@ export class ApplicationService extends RecordDataModelService<Application> {
         .createQueryBuilder("application")
         .select([
           "application.id",
+          "currentAssessment.id",
```

Comment:
This is the PIR fix. The missing ID was making type think that a new assessment was supposed to be created.

Reply:
mm, what was the issue in PIR?